infrastructure/builder/src/build/index.js

```diff
@@ -55,6 +55,7 @@ class Build {
       return;
     }
     const context = await taskgraph.run({'build-can-start': true});
+    console.log('DONE');
```

Comment:
Is this stray?

Reply:
I was just testing you?  Yeah, thanks :)